nvpy/view.py

```diff
@@ -440,12 +441,24 @@ def __init__(self, master, font_family, font_size, config: NotesListConfig):
         yscrollbar.pack(side=tk.RIGHT, fill=tk.Y)
 
         f = tkFont.Font(family=font_family, size=font_size)
+
+        # Calculate the text width in characters based on the font
+        # and the desired pixel width of the frame, if the pixel_width
+        # is set at all.
+        text_width = 30
+        try:
+            if config.pixel_width:
+                sentence = 'MmMmMmMmMmMmMmMmMmMmMmMmMmMmMm'
+                text_width = int(float(config.pixel_width)/((float(f.measure(sentence)))*.64)*len(sentence))
```

Comment:
@yuuki0xff I really hate this calculation. I had to add the *.64 on my machine to get the pixel width calculated to be anywhere near what the vale is saved to in the file from one time opening the app to the next. If you know a better way to figure out how wide it should be (or how to make the Frame width be what this follows), I'm all for it. That, or take  out this try/except, and let the default just be what it is on start.

Reply:
You can specify width and height in pixels by using Frame. See https://stackoverflow.com/a/14897186 for details.

Changing the initial size of the Frame should work.
https://github.com/cpbotha/nvpy/blob/272f6dfd771975b8a041f8c9e81733a8beead4ec/nvpy/view.py#L1417